drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/aggregated/aggregated_import_concern.rb

```diff
@@ -0,0 +1,34 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module HmisCsvTwentyTwenty::Aggregated::AggregatedImportConcern
+  extend ActiveSupport::Concern
+
+  included do
+    def find_matching_record
+      self.class.find_by(self.class.hud_key => self[self.class.hud_key], data_source_id: data_source_id)
+    end
+
+    def newer_than?(matching_record)
+      false unless matching_record.DateUpdated >= self.DateUpdated
+
+      source_hash != matching_record.source_hash
+    end
+
+    def self.import(batch)
+      updated_batch = []
+      batch.each do |record|
```

Comment:
can we rename `record` to `incoming` and `matching_record` to `existing` or something similar.